src/neuroconv/datainterfaces/ecephys/openephys/openephysbinarydatainterface.py

```diff
@@ -86,15 +92,19 @@ def __init__(
         available_streams = self.get_stream_names(folder_path=folder_path)
         if len(available_streams) > 1 and stream_name is None:
             raise ValueError(
-                "More than one stream is detected! Please specify which stream you wish to load with the `stream_name` argument. "
-                "To see what streams are available, call `OpenEphysRecordingInterface.get_stream_names(folder_path=...)`."
+                "More than one stream is detected! "
+                "Please specify which stream you wish to load with the `stream_name` argument. "
+                "To see what streams are available, call "
```

Comment:
I think you should print the streams for users here given that you already calculated them.

Reply:
Feel free to add this is a separate PR, I probably won't in this one (I was just breaking up a line length violation in this change)

Reply:
Makes sense.